src/main/java/com/gojek/beast/worker/Worker.java

```diff
@@ -24,6 +26,11 @@ public void run() {
         do {
             status = job();
         } while (!state.isStopped() && status.isSuccess());
+
+        if (!status.isSuccess()) {
+            statsClient.increment("global.errors,exception=" + status.toString());
```

Comment:
Status.toString would not give you the class of exception, it would be a string containing the details
```
"FailureStatus{"
                + "exception=" + cause.getClass().getName()
                + ", cause=" + cause.getCause()
                + ", message='" + ((message != null) ? message : cause.getMessage()) + '\'' + '}';
```

Reply:
I thought of posting the entire string in the exception to have details. we can also do `status.getException()` what do you think?